test/simplegraphs/generators/staticgraphs.jl

```diff
@@ -536,5 +536,19 @@
         @test_throws DomainError lollipop_graph(0, 1)
         @test_throws DomainError lollipop_graph(-1, -1)
     end
+   @testset "Friendship Graphs" begin
+        # the Friendship Graphs is connected graph consist of n cycle of lenght 3 share only one node 
+        for n in [10, 15, 20]
+         g=friendship_graph(n)
+         @test length(connected_components(g)) == 1
+         @test articulation(g) == [1]
```

Comment:
articulation is a strange test. Why not test the number of edges and vertices, and – if you want to get really detailed, ensure that the outer edges of the triangle exist?

Reply:
i just want to make sure the shared node between all the triangles is 1

Reply:
I think it would be easier to verify that vertex 1 has degree 2n.